pynest/nest/server/hl_api_server.py

```diff
@@ -212,7 +212,7 @@ def get_restricted_globals():
     def getitem(obj, index):
         if obj is not None and type(obj) in (list, tuple, dict, nest.NodeCollection):
             return obj[index]
-        msg = f"Error while getting restricted globals: unidentified object '{obj}'."
+        msg = "Error while getting restricted globals: unidentified object '{}'.".format(obj)
```

Comment:
We changed it because f-strings are only compatible with Python 3.6 on. If support from NEST is only intended from this version of Python on, then we can change it back.

Reply:
NEST master currently requires (although somewhat loosely) Python 3.6 or newer. Also, with #1954 it will enforce a requirement of Python 3.8 or newer.